From: Optical Media Tools Team <pkg-opt-media-team@lists.alioth.debian.org>
Date: Sun, 8 Apr 2012 21:51:29 +0900
Subject: Disables to skip on encrypted disks (e.g. DVD with CSS-Encryption).

Description: Disables to skip on encrypted disks (e.g. DVD with CSS-Encryption).
Author: n/a
Last-Update: 2012-04-08
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
---
 scsi-layer.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/scsi-layer.c b/scsi-layer.c
index cad13d4..866b7ac 100644
--- a/scsi-layer.c
+++ b/scsi-layer.c
@@ -2695,12 +2695,13 @@ Image* OpenImageFromDevice(char *device)
        	 return NULL;
       }
    }
-
+/*
    if(dh->mainType == DVD && query_copyright(dh))
    {  CloseImage(image);
       Stop(_("This software does not support encrypted media.\n"));
       return NULL;
    }
+*/
 
    /* Create the bitmap of simulated defects */
 
